core/domain/feedback_services_test.py

```diff
@@ -400,6 +401,33 @@ def test_get_thread_summaries(self):
             # Check if the summaries match.
             self.assertEqual(thread_summary, thread_summaries[index])
 
+    def test_get_thread_summaries_load_test(self):
+        # The speed of fetching the summaries of 100 threads having 5 messages
+        # should be less than 8 seconds.
```

Comment:
Yes, that seems reasonable. Try to get it to 0.5 secs, because this is measuring just the query execution time. There's going to be additional overhead from the learner's perspective in terms of the round-trip time to the server, additional backend latency from other sources, etc.